Resources/Textures/SS220/Structures/barricades.rsi/meta.json

```
@@ -0,0 +1,47 @@
+{
+  "version": 1,
+  "license": "EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt",
+  "copyright": "Sprited by EstKemran (Github) for SS220"
```

```suggestion
  "copyright": "Sprited by EstKemran (Github) for SS220",
```
падают тесты из-за отсутствия запятой